lib/terraforming/resource/iam_group_membership.rb

```diff
@@ -16,24 +16,24 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_group_membership")
+        apply_template(@client, 'tf/iam_group_membership')
       end
 
       def tfstate
         iam_groups.inject({}) do |resources, group|
           membership_name = membership_name_of(group)
 
           attributes = {
-            "group"=> group.group_name,
-            "id" => membership_name,
-            "name" => membership_name,
-            "users.#" => group_members_of(group).length.to_s,
+            'group' => group.group_name,
+            'id' => membership_name,
+            'name' => membership_name,
+            'users.#' => group_members_of(group).length.to_s,
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)